theorems/T000652.md

```diff
@@ -0,0 +1,9 @@
+---
+uid: T000652
+if:
+  P000208: true
+then:
+  P000130: true
+---
+
+For each $x \in X$, the set of neighborhoods containing $X$ is a local basis of compact neighborhoods, since every open set is compact.

```

Comment:
```suggestion
For each $x \in X$, the set of open neighborhoods of $x$ is a local basis of compact neighborhoods, since every open set is compact.

```